content/hi/docs/reference/glossary/configmap.md

```diff
@@ -0,0 +1,16 @@
+---
+title: ConfigMap
```

Comment:
Translate this please

Reply:
You can use `aka:` to also provide the original name.